consumergroup.go

```diff
@@ -860,8 +861,7 @@ func (cg *ConsumerGroup) coordinator() (coordinator, error) {
 	if err != nil {
 		return nil, err
 	}
-
-	address := fmt.Sprintf("%v:%v", out.Coordinator.Host, out.Coordinator.Port)
+	address := net.JoinHostPort(out.Coordinator.Host, fmt.Sprintf("%d", out.Coordinator.Port))
```

Comment:
2 things, could we re-add the newline that was here before? And also use `strconv.Itoa` instead of `fmt.Sprintf("%d"`
```suggestion
	address := net.JoinHostPort(out.Coordinator.Host, strconv.Itoa(out.Coordinator.Port))
```